test/fuzzing/status_fuzz.cpp

```diff
@@ -16,6 +16,7 @@
 #include "interfaces/iroha_internal/transaction_batch_factory_impl.hpp"
 #include "interfaces/iroha_internal/transaction_batch_parser_impl.hpp"
 #include "module/irohad/ametsuchi/ametsuchi_mocks.hpp"
+#include "module/irohad/ametsuchi/mock_tx_presence_cache.hpp"
```

Comment:
I would suggest to revert the changes in this file since it does not compile, and fix it in another pull request.